ts/features/payments/bizEventsTransaction/screens/PaymentsTransactionBizEventsListScreen.tsx

```diff
@@ -171,31 +195,58 @@ const PaymentsTransactionBizEventsListScreen = () => {
 
   const renderLoadingFooter = () => (
     <>
-      {isLoading &&
-        Array.from({ length: 5 }).map((_, index) => (
-          <ListItemTransaction
-            isLoading={true}
-            key={index}
-            transactionStatus="success"
-            transactionAmount=""
-            title=""
-            subtitle=""
-          />
-        ))}
-      {!isLoading && !continuationToken && <ShowLegacyTransactionsButton />}
+      {isLoading && (
```

Comment:
Let's consider refactoring by moving some components and their logic outside the main screen to improve readability, simplify the code, and make future updates easier

Reply:
Let me know what do you think about the small refactor addressed in this commit: 5c3579153abf9ceef9ee6b69fe44fcf58b254ddf

Reply:
Great job!